Src/PerformanceCollector/Filtering/CollectionConfiguration.cs

```diff
@@ -0,0 +1,327 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Filtering
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.QuickPulse.Helpers;
+
+    internal class CollectionConfiguration
+    {
+        private readonly CollectionConfigurationInfo info;
+
+        private readonly List<OperationalizedMetric<RequestTelemetry>> requestTelemetryMetrics = new List<OperationalizedMetric<RequestTelemetry>>();
+
+        private readonly List<OperationalizedMetric<DependencyTelemetry>> dependencyTelemetryMetrics =
+            new List<OperationalizedMetric<DependencyTelemetry>>();
+
+        private readonly List<OperationalizedMetric<ExceptionTelemetry>> exceptionTelemetryMetrics =
+            new List<OperationalizedMetric<ExceptionTelemetry>>();
+
+        private readonly List<OperationalizedMetric<EventTelemetry>> eventTelemetryMetrics = new List<OperationalizedMetric<EventTelemetry>>();
+
+        private readonly List<OperationalizedMetric<TraceTelemetry>> traceTelemetryMetrics = new List<OperationalizedMetric<TraceTelemetry>>();
+
+        private readonly List<OperationalizedMetric<MetricValue>> metricMetrics = new List<OperationalizedMetric<MetricValue>>();
+
+        private readonly List<Tuple<string, AggregationType>> telemetryMetadata = new List<Tuple<string, AggregationType>>();
```

Comment:
Done.
I've added a remarks section at the top, and split member fields into two regions based on their usage.